flexmeasures/ui/templates/crud/asset.html

```diff
@@ -7,38 +7,44 @@
 
 
 {% block divs %}
-<nav aria-label="breadcrumb">
-  <ol class="breadcrumb">
-    {% for breadcrumb in breadcrumb_info["ancestors"] %}
-      <li class="breadcrumb-item{% if loop.last %} dropdown active{% endif %}" {% if loop.last %}aria-current="page"{% endif %}>
-        {% if breadcrumb["url"] is not none and not loop.last %}
-          <a href="{{ breadcrumb['url'] }}">{{ breadcrumb['name'] }}</a>
-        {% else %}
-          <a class="dropdown-toggle" data-toggle="dropdown" href="#">{{ breadcrumb['name'] }}<span class="caret"></span></a>
-          <ul class="dropdown-menu">
-            {% for sibling in breadcrumb_info["siblings"] %}
-              <li><a href="{{ sibling['url'] }}">{{ sibling["name"] }}</a></li>
-            {% endfor %}
-          </ul>
-        {% endif %}
-      </li>
-    {% endfor %}
-  </ol>
+
+<nav aria-label="breadcrumb ">
+    <ol class="breadcrumb p-2">
+        {% for breadcrumb in breadcrumb_info["ancestors"] %}
+        <li class="breadcrumb-item{% if loop.last %} dropdown active{% endif %}" {% if loop.last %}aria-current="page"{% endif %}>
+            {% if breadcrumb["url"] is not none and not loop.last %}
+            <a  style="text-decoration:none;" href="{{ breadcrumb['url'] }}">{{ breadcrumb['name'] }}</a>
+            {% else %}
+            <a style="text-decoration:none;" href="{{ breadcrumb['url'] }}" class="dropdown-toggle" data-bs-toggle="dropdown"  aria-expanded="false">{{ breadcrumb['name'] }}
```

Comment:
The docs recommend to add  `role="button"` here if the main dropdown element is not a button.